pkg/userinfo/roleRef_test.go

```diff
@@ -407,3 +409,287 @@ func Test_getRoleRefByClusterRoleBindings(t *testing.T) {
 		})
 	}
 }
+
+type roleBindingLister struct {
+	ret []*rbacv1.RoleBinding
+	err error
+}
+
+func (l roleBindingLister) List(labels.Selector) ([]*rbacv1.RoleBinding, error) {
+	return l.ret, l.err
+}
+
+type clusterRoleBindingLister struct {
+	ret []*rbacv1.ClusterRoleBinding
+	err error
+}
+
+func (l clusterRoleBindingLister) List(labels.Selector) ([]*rbacv1.ClusterRoleBinding, error) {
+	return l.ret, l.err
+}
+
+func TestGetRoleRef(t *testing.T) {
+	type args struct {
+		rbLister  RoleBindingLister
+		crbLister ClusterRoleBindingLister
+		userInfo  authenticationv1.UserInfo
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    []string
```

Comment:
I think naming these a bit more clearly could help e.g.:
`wantRoles` and `wantClusterRoles`

Tests are more difficult to read than necessary currently IMO.

Reply:
@MarcelMue fixed.